README.md

```diff
@@ -49,7 +49,7 @@ promptly identifying and addressing issues that impact service level objectives.
 
 Apply `slo: "true"` under labels section of any alerting rule.
   ```
-  labels:
+  annotations:
```

Comment:
Should be `labels`
See https://github.com/redhat-appstudio/o11y/blob/main/rhobs/alerting/data_plane/prometheus.application_alerts.yaml#L18

Reply:
not sure what went wrong here, but should be fixed now